proxy/src/main/java/org/apache/accumulo/proxy/Proxy.java

```diff
@@ -135,7 +136,8 @@ public static TServer createProxyServer(Class<?> api, Class<?> implementor, fina
     @SuppressWarnings("unchecked")
     Constructor<? extends TProcessor> proxyProcConstructor = (Constructor<? extends TProcessor>) proxyProcClass.getConstructor(proxyIfaceClass);
 
-    final TProcessor processor = proxyProcConstructor.newInstance(RpcWrapper.service(impl));
+    @SuppressWarnings({"rawtypes", "unchecked"})
```

Comment:
Neither rawtypes, nor unchecked are necessary to suppress here if we cast to `TBaseProcessor<?>`.